apps/scorecard/server/main.js

```diff
@@ -215,201 +246,240 @@ const doPlotParams = function () {
                 options: [],
                 controlButtonCovered: true,
                 default: new Date().toLocaleDateString(),
-                controlButtonVisibility: 'block',
+                controlButtonVisibility: 'none',
```

Comment:
Please revert this to "block". The hideOtherFor will take care of it. The reason it isn't working is because there was a duplicate line, as noted earlier.

Reply:
good catch